tpcx_bb/queries/q08/tpcx_bb_query_08_sql.py

```diff
@@ -217,8 +223,12 @@ def main(data_dir, client, bc, config):
         get_unique_sales_keys_from_sessions, review_cat_code=REVIEW_CAT_CODE
     )
 
+    del merged_df
```

Comment:
Does this do anything here? I believe Dask can't delete these references until the various map partitions calls actually get computed in lines 230/231

Reply:
@Christian8491 if this is removed, does it affect the query success?

Reply:
Let me just try if all is ok with his validation.

Reply:
Just tested at sf **1K** on **dgx201**. cc @beckernick 

```
(bsql-tpcx) christianc@rl-dgx2-d17-u05-rapids-dgx201:~/tpcx-bb/tpcx_bb/queries/q08$ python tpcx_bb_query_08_sql.py --config_file=/home/christianc/yaml_dir/validation_1k_config.yaml
Using default arguments
Connected to tcp://10.150.162.27:8786
BlazingContext ready
/home/christianc/miniconda3/envs/bsql-tpcx/lib/python3.7/site-packages/dask_cudf/core.py:329: UserWarning: Rearranging data by column hash. Divisions will lost. Set ignore_index=False to preserve Index values.
  "Rearranging data by column hash. Divisions will lost. "
Standard ETL Query
/home/christianc/miniconda3/envs/bsql-tpcx/lib/python3.7/site-packages/pandas/util/__init__.py:12: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.
  import pandas.util.testing
/home/christianc/miniconda3/envs/bsql-tpcx/lib/python3.7/site-packages/cudf/tests/utils.py:83: FutureWarning: The 'check_less_precise' keyword in testing.assert_*_equal is deprecated and will be removed in a future version. You can stop passing 'check_less_precise' to silence this warning.
  tm.assert_frame_equal(left, right, **kwargs)
Correctness Assertion True
```

Reply:
thanks!